src/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
```suggestion
<html lang="ru">
```
Основной язык страницы -- русский